src/TopBar/index.less

```diff
@@ -40,7 +41,6 @@
   .text--regular;
   .text--truncate;
   color: @neutral_white;
-  font-style: italic;
```

Comment:
Just wondering, was this change also related to accessibility?

Reply:
This wasn't flagged by the accessibility audit. This is just a design tweak that the design team asked us to make while changing the color.